src/main/scala/io/qbeast/spark/utils/Params.scala

```diff
@@ -40,3 +40,7 @@ object MetadataConfig {
   final val replicatedSet = "qbeast.replicatedSet"
   final val lastRevisionID = "qbeast.lastRevisionID"
 }
+
+object Staging {
```

Comment:
Do we need to create an object for this? Or could we place it in another existing class like `QbeastDeltaSnapshot`? 
Because probably each format would have their own isStagingFile predicate.